web/src/user_groups_settings_ui.js

```diff
@@ -179,6 +180,31 @@ export function change_state(section) {
     show_user_group_settings_pane.nothing_selected();
 }
 
+export let new_user_group_can_manage_group_widget = null;
+
+function dropdown_setup() {
+    new_user_group_can_manage_group_widget = new dropdown_widget.DropdownWidget({
+        widget_name: "new_user_group_can_manage_group",
+        get_options: () => user_groups.get_dropdown_options_for_can_manage_group_setting(true),
+        item_click_callback(event, dropdown) {
+            dropdown.hide();
+            event.preventDefault();
+            event.stopPropagation();
+            new_user_group_can_manage_group_widget.render();
+        },
+        $events_container: $("#groups_overlay"),
+        tippy_props: {
+            placement: "bottom-start",
+        },
+        on_mount_callback(dropdown) {
+            $(dropdown.popper).css("min-width", "300px");
+        },
+        default_text: $t({defaultMessage: "No user groups"}),
+        default_id: people.get_user_system_group_name_from_user_id(people.my_current_user_id()),
+        unique_id_type: dropdown_widget.DATA_TYPES.STRING,
```

Comment:
I have to use data type as `string` here because with `number` there was problem with options `All members of this group` and `You`. So, now `unique_id` for role based system groups will be `group_id` only but in the form of `string` rather than `number`.